### PR TITLE
Release/tuesday

### PR DESCRIPTION
**Pull request by kyranjamie**
> Try out this version of Leather — [download extension builds](https://github.com/leather-wallet/extension/actions/runs/6469744247).

- https://github.com/leather-wallet/extension/issues/4223
- https://github.com/leather-wallet/extension/issues/3889

### CODE REVIEW

**Review by fbwoolf — APPROVED** [review]

**Review by alter-eggo — APPROVED** [review]
there are two commits with same text `fix: outdated version warning`

File: src/app/components/fees-row/fees-row.tsx [hunk 1]
```diff
@@ -46,14 +49,27 @@ export function FeesRow(props: FeeRowProps): React.JSX.Element {
...
   useEffect(() => {
-    if (hasFeeEstimates && !feeField.value && !isCustom) {
+    if (props.defaultFeeValue) {
+      feeHelper.setValue(
+        convertAmountToBaseUnit(
+          new BigNumber(Number(props.defaultFeeValue)),
+          STX_DECIMALS
+        ).toString()
+      );
+      feeTypeHelper.setValue(FeeTypes[FeeTypes.Custom]);
```
Comment (pete-watters) [inline src/app/components/fees-row/fees-row.tsx:59]: Maybe we can fix these warnings marking as `void`

File: src/app/components/nonce-setter.tsx [hunk 2]
```diff
@@ -1,15 +1,12 @@
...
 
 import { StacksSendFormValues, StacksTransactionFormValues } from '@shared/models/form.model';
 
 import { useNextNonce } from '@app/query/stacks/nonce/account-nonces.hooks';
 
-interface NonceSetterProps {
-  children: ReactNode;
-}
-export function NonceSetter({ children }: NonceSetterProps) {
+export function NonceSetter() {
```
Comment (pete-watters) [inline src/app/components/nonce-setter.tsx:9]: This is returning nothing now so we could move the code into `src/app/features/stacks-transaction-request/stacks-transaction-signer.tsx`

File: src/background/messaging/rpc-methods/sign-stacks-transaction.ts [hunk 3]
```diff
@@ -0,0 +1,188 @@
+import { RpcErrorCode } from '@btckit/types';
```
Comment (pete-watters) [inline src/background/messaging/rpc-methods/sign-stacks-transaction.ts:1]: This is a huge file that could be split up

**Review by pete-watters — APPROVED** [review]
LGTM just left some style nits

**Comment by kyranjamie** [comment]
> there are two commits with same text fix: outdated version warning

That was my bad, but too late, it's in main and don't want to force push.

@pete-watters legit feedback, but also too late. Feel free to open issues/address these

**Pull request merged**